packages/mad-tag-ocr/src/components/Camera/OCRCamera.tsx

```diff
@@ -199,7 +199,7 @@ export const OCRCamera = ({
                     frameProcessor={frameProcessor}
                     enableZoomGesture
                     videoStabilizationMode="auto"
-                    orientation="portrait"
+                    outputOrientation="portrait"
```

Comment:
This has a red line in my vscode. However, I'm not even able to run `yarn install`, so that might be the reason why.

Did the props change? I see there's a new (non-major) version from 2 weeks ago

Reply:
When I was able to install, the red line was removed. This comment can be resolved